src/mono/wasm/build/WasmApp.targets

```diff
@@ -155,6 +155,21 @@
 
     <WasmRuntimeAssetsLocation Condition="'$(WasmRuntimeAssetsLocation)' == ''">_framework</WasmRuntimeAssetsLocation>
     <BlazorCacheBootResources Condition="'$(BlazorCacheBootResources)' == ''">true</BlazorCacheBootResources>
+
```

Comment:
Blazor specific props can be omitted here
- BlazorCacheBootResources
- BlazorEnableTimeZoneSupport